linera-core/src/client/mod.rs

```diff
@@ -3352,6 +3352,19 @@ impl<Env: Environment> ChainClient<Env> {
         }
     }
 
+    /// Returns the indices and events from the storage
+    pub async fn events_from_index(
```

Comment:
This can OOM, right? There's no backpressure, paging etc

Reply:
Yes, this can OOM. There are ways to get GraphQL pagination, but we have not implemented that thus far.